desktop/app.js

```diff
@@ -162,7 +162,7 @@ function createMenuTemplate( settings ) {
 			type: 'separator'
 		} );
 		fileMenu[ 'submenu' ].push( {
-			label: 'Exit',
+			label: '&Exit',
```

Comment:
Just reviewing this now, sorry it took so long! I think `x` is more commonly used in `Exit`.